conf/application.conf

```diff
@@ -91,9 +91,11 @@ play.modules.enabled += "config.DIModule"
 
 # Api related config
 api {
+  documentation-url = "https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-accounts-api"
```

Comment:
Small thing here, can you move documentation-url to the end of the `api` object. Just a visual thing and consistent with other apis